babel.config.js

```diff
@@ -1,3 +1,12 @@
 module.exports = {
-  presets: [['@shopify/babel-preset', {typescript: true, react: true}]],
+  presets: [
+    [
+      '@shopify/babel-preset',
+      {
+        typescript: true,
+        react: true,
+        reactOptions: {runtime: 'automatic', useBuiltIns: true},
```

Comment:
Reading https://babeljs.io/docs/en/babel-preset-react/#react-classic-runtime it suggests that `useBuiltIns` is only a valid option when using the classic runtime. It has no effect when using the automatic runtime. Thus there's no value in specifying it here.

```suggestion
        reactOptions: {runtime: 'automatic'},
```